velox/connectors/hive/HiveSessionProperty.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <optional>
+#include <string>
+#include "PropertyMetadata.h"
+#include "velox/connectors/hive/HiveConfig.h"
+#include "velox/core/Config.h"
+
+using namespace facebook::velox::common;
+namespace facebook::velox::connector::hive {
+
+/// Hive connector configs.
+class HiveSessionProperty {
```

Comment:
Can we merge HiveSessionProperty into hive config at Prestissmo? The session property is Prestissimo specific things. Let's keep it transparent to velox? Thanks!